source/ui/AMBeamlineCameraWidget.cpp

```diff
@@ -139,6 +163,13 @@ QPointF AMBeamlineCameraWidget::crosshairPosition() const
 void AMBeamlineCameraWidget::setCrosshairPosition(const QPointF &pos) const
 {
 	videoWidget_->setCrosshairPosition(pos);
+	recenterCrosshairPushbutton_->setToolTip(QString("x: %1, y: %2").arg(pos.x()).arg(pos.y()));
+}
+
+void AMBeamlineCameraWidget::setCrosshairCenterPosition (const QPointF &pos)
+{
+	crosshairCenter_.setX(pos.x());
```

Comment:
This could just be replaced by `crosshairCenter_ = pos;`.  You don't need to change it, just pointing it out.